x/auth/keeper/keeper_test.go

```diff
@@ -64,7 +64,7 @@ func (suite *KeeperTestSuite) SetupTest() {
 
 	suite.msgServer = keeper.NewMsgServerImpl(suite.accountKeeper)
 	queryHelper := baseapp.NewQueryServerTestHelper(suite.ctx, suite.encCfg.InterfaceRegistry)
-	types.RegisterQueryServer(queryHelper, suite.accountKeeper)
+	types.RegisterQueryServer(queryHelper, keeper.NewQuerier(suite.accountKeeper))
```

Comment:
We need to call `authv1.RegisterQueryServer` here too. The struct wrapping is only necessary if there are keeper method conflicts, but generally doesn't hurt.